### PR TITLE
[MDS-4928] Assign initial status in incident creation in Core

### PR DESCRIPTION
**Pull request by areyeslo**
## Objective 

[MDS-4928](https://bcmines.atlassian.net/browse/MDS-4928)

https://bcmines.atlassian.net/browse/MDS-4970
https://bcmines.atlassian.net/browse/MDS-4928

- Replaced initial status code from `WNS`= Written Notice Submitted to any of the status codes `AFR` - Awaiting Final Report or `FRS` - FInal Report Submitted depending on the existence of Final Report.
-  `WNS` status code is not longer used. However, it is left in the database.
- The dangerous occurrence determined by the Mine is removed in Minespace-web and Core-web. Just the DO determined by the inspector is present.
- Uploading final report is optional in Minespace-web and core-web, can be done after submission (this would create an incident with initial status `AFR`.

### CODE REVIEW

File: services/core-web/src/components/mine/Incidents/MineIncident.js [hunk 1]
```diff
@@ -125,6 +126,36 @@ export class MineIncident extends Component {
...
+      ].includes(this.props.formValues.determination_type_code)
+    ) {
+      determinationTypeCode = false;
+    } else {
+      determinationTypeCode = this.props.formValues.mine_determination_type_code;
+    }
+
+    if (!this.props.formValues.status_code) {
+      if (!determinationTypeCode) {
+        this.props.formValues.status_code = "WNS";
```
Comment (JustinMacaulay) [inline services/core-web/src/components/mine/Incidents/MineIncident.js]: What are the possible statuses for an incident that is marked as a non dangerous occurrence? Are they always WNS and then Closed?

**Review by JustinMacaulay — COMMENTED** [review]
just one question around status

File: services/core-web/src/components/mine/Incidents/MineIncident.js (hunk 1 above)
Comment (areyeslo) [reply services/core-web/src/components/mine/Incidents/MineIncident.js]: That's correct., they are created as WNS and then go to the next stages which I guess are: Reviewing Severity Status/ Information Missing Status -> Under Review -> Closed.

Comment (areyeslo) [reply services/core-web/src/components/mine/Incidents/MineIncident.js]: The logic completely changed. There is no WNS status code, but `AFR` or `FRS` code depending on the existence of the Final Report.

**Review by matbusby-fw — DISMISSED** [review]

**Review by matbusby-fw — APPROVED** [review]

**Pull request merged**